### PR TITLE
content(docs): Add section on dokku for deploiement on polytech servers

### PR DESCRIPTION
**Pull request by BrokenSwing**

### CODE REVIEW

**Comment by BrokenSwing** [comment]
@Axel-Duval Could you review ? Tell me if you see something that would be missing 😉

File: content/docs/deploiement/dokku.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+contributors: ["Florent Hugouvieux"]
+---
+
+## Qu'est-ce que Dokku
+
+[Dokku](https://dokku.com/) est un outil de PaaS (Platform as a Service) auto-hébergé construit sur Docker qui vous permet de déployer vos applications web. Dokku est grandement inspiré d'[Heroku](https://www.heroku.com/) qui est un service en ligne faisant aussi du PaaS, ainsi, vous pouvez utiliser tout [buildpack](https://devcenter.heroku.com/articles/buildpacks) supporté par [herokuish](https://github.com/gliderlabs/herokuish) directement sur une instance Dokku.
+
+## Déployer sur Dokku à Polytech
+
+En tant qu'étudiant en informatique et gestion à Polytech Montpellier, vous pouvez avoir accès à une instance de Dokku déployée directement sur les serveurs de Polytech Montpellier.
```
Comment (Axel-Duval) [inline content/docs/deploiement/dokku.md:22]: sur les serveurs de l'école (éviter répétition)

File: content/docs/deploiement/dokku.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Pour avoir accès à une instance, vous devez [contacter Luca Cimini sur le Mattermost](https://mattermost.polytech.umontpellier.fr/ig/messages/@luca.cimini) et lui envoyer votre **clé publique** SSH afin qu'il puisse l'ajouter à la machine virtuelle correspondant à votre année :
+
+- IG3 : cluster-ig3.igpolytech.fr
+- IG4 : cluster-ig4.igpolytech.fr
+- IG5 : cluster-ig5.igpolytech.fr
+
+Une fois votre clés publique SSH ajoutée sur le cluster correspondant à votre année, vous pouvez accéder à l'instance Dokku par SSH sur ce dernier. Exemple, si je suis en IG3 :
```
Comment (Axel-Duval) [inline content/docs/deploiement/dokku.md:30]: clé

**Review by Axel-Duval — COMMENTED** [review]
Some typos

**Pull request merged**